### DISCUSSION TITLE
Relative variance scaling.

### DISCUSSION THREAD

**Discussion by jackcturner** [astromatic/sextractor#70] [Q&A]
Hello, 

I'm trying to understand the Source Extractor approach to weighting, but I'm struggling to understand the conversion from relative to absolute variance. The manual states that "A robust scaling to the appropriate absolute level is then performed by comparing this variance map to an internal, low-resolution, absolute variance map built from the science image itself", but it's not clear to me what this "comparison" would be.

Any help would be very much appreciated!

**Comment by Eririf** [comment]
Typically , `absolute variance map built from the science image itself` is the background rms map computed by sextractor
